internal/jimm/juju/jimm_test.go

```diff
@@ -92,30 +96,49 @@ func TestListControllers(t *testing.T) {
 		jimmAdmin           bool
 		expectedControllers []dbmodel.Controller
 		expectedError       string
-	}{{
-		about:     "superuser can list controllers",
-		user:      env.User("alice@canonical.com").DBObject(c, j.Database),
-		jimmAdmin: true,
-		expectedControllers: []dbmodel.Controller{
-			env.Controller("test1").DBObject(c, j.Database),
-			env.Controller("test2").DBObject(c, j.Database),
-			env.Controller("test3").DBObject(c, j.Database),
+	}{
```

Comment:
can we add a test to check the user has this permission if `user:*` (everybody) has this permission